packages/atclient/src/atclient.c

```diff
@@ -261,7 +268,6 @@ int atclient_pkam_authenticate(atclient *ctx, const char *atsign, const atclient
   /*
    * 5. Get atserver_host and atserver_port
    */
-  bool should_free_atserver_host;
```

Comment:
Moved up, as it was undefined in some scenarios due to goto statements, there are a couple of other instances I caught.

Reply:
Good catch